tests/integration/scripts/inotify_file.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
```

Comment:
For inotify maybe using inotifywait in the script?

Reply:
Agreed, much simpler to do it using scripts... tks for the suggestion.